x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -135,17 +142,19 @@ export const AlertIndex = memo(() => {
 
       if (columnId === 'alert_type') {
         return (
-          <Link
+          <EuiLink
             data-testid="alertTypeCellLink"
-            to={urlFromQueryParams({ ...queryParams, selected_alert: 'TODO' })}
+            onClick={() =>
```

Comment:
missing `href`??

Reply:
does it require an `href`? we're getting all the functionality we need from just the onclick since its just changing the search query in the url

Reply:
I think the href is needed if you would like to enable users to open the link in a separate tab/window or be able to copy the link

Reply:
ok, we've opened a ticket for it to address

Reply:
oh geez